Azure/sonic-snmpagent
# [Namespace]: Fix interface counters in RFC 1213

Signed-off-by: SuvarnaMeenakshi <sumeenak@microsoft.com>



**- What I did**
In multi-asic platform, SAI OID is not unique for the whole device. It is unique for an asic and within a single namespace.
This PR is to fix InterfacesMIB rfc1213 implementation to make sure that interfaces counters is keyed based on interface index and not SAI OID.

**- How I did it**
- Fix library functions to use namespace:sai-oid as key, instead of using sai-oid.
- Fix RFC 1213 to use the new key format.

**- How to verify it**
Test InterfacesMIB on single and multi-asic platform.
single-asic platform should see no change in output.
multi-asic platform should show right interface counter values.

**- Description for the changelog**
